proud-puffins/djangoProject/earlydating/decorators.py

```diff
@@ -0,0 +1,44 @@
+from functools import wraps
+from django.http import HttpResponse
+from django.shortcuts import redirect
+
+
+#  To avoid double login/register if you are logged in
```

Comment:
Be sure to include your documentation for a given function as a docstring, which is a string (typically using triple quotes) as the first line below the function signature (the `def` part). Docstrings for functions have a number of advantages over regular comments.

```
def unauthenticated_user(view_func):
    """To avoid double login/register if you are logged in"""
    ...
```

This is what you'd want to do.